config/special_users.json

```diff
@@ -7,5 +7,6 @@
     "U01230GJXS6",
     "UE8DH0UHM",
     "U015MCCBXBP",
-    "U011CFN98K1"
+    "U011CFN98K1",
+    "USLACKBOT"

```

Comment:
Adding slackbot has the potential to cause abuse - we shouldn't let it do that. Please remove this item from the list.

Reply:
![request](https://user-images.githubusercontent.com/44281159/87844835-3f2f9300-c876-11ea-8010-31b65949cbfd.png)
I was requested by a dev to add slackbot to the list.

Reply:
Sorry, that's clearly not slack, much less our official slack. Would you be willing to provide a direct screenshot of the file within slack, so we know it isn't falsified? Perhaps a link would help as well.

Reply:
![slack](https://user-images.githubusercontent.com/44281159/87844941-093ede80-c877-11ea-8325-ee102a2110f0.png)
OFC - I am using a third party client called RipCord in order to speed slack up for me. While I am not allowed to post the link due to it being a private channel in a private slack (and quite frankly as a fellow dev you know this), I did take a screenshot. Please excuse the light mode - I do not use the slack web app often.

Reply:
As the dev who wrote the above message, I am well aware of what it said - your modifications here are frankly not helping the situation. As much as we appreciate these PRs there doesn't seem to be a super valid reason for it. Closing until we can find said reason.

Reply:
Quite frankly I am astonished that you are claiming there are modifications - please feel free to DM me in the slack about this. For now I agree that it is no longer productive to comment on this issue.